.github/workflows/build.yml

```diff
@@ -84,6 +84,8 @@ jobs:
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock', '**/package.json') }}
+      - name: install dependencies
```

Comment:
The issue resides in the node modules cache keys that is different between lint and build. We should be using the same cache key, my bad !